index.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
       if (google.key) {
         params.push('key=' + encodeURIComponent(google.key));
       }
+      if (typeof google.libraries !== 'undefined' && google.libraries.length > 0) {
+        params.push('libraries=' + google.libraries.join());
```

Comment:
Actually, calling **join()** puts in the commas by default.

Escaping is a good idea, though.